src/saltext/vmware/states/storage_policy.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+import logging
+
+import saltext.vmware.utils.connect as connect
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "vsphere_storage_policy"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def storage_policy(name, storagePolicies):
```

Comment:
**suggestIon** prefer snake_case, i.e. `s/storagePolicies/storage_policies`

Reply:
Renamed to policies